hail/python/test/hail/vds/test_vds.py

```diff
@@ -826,6 +827,14 @@ def test_split_sparse_roundtrip():
     assert vds2.reference_data._same(vds_split.reference_data.drop('ref_allele'))
 
 
+def test_split_alters_ref_gt():
```

Comment:
I think this test only follows one path. Can you make sure both paths are traced or tell me I'm wrong?